### DISCUSSION TITLE
最新版本（1.6.1）为什么没办法解析视频？

### DISCUSSION THREAD

**Discussion by FFFFFIR** [leiurayer/downkyi#980] [Q&A]
解析视频时地址栏会转圈圈，转完以后还是没有文件信息，也不可以下载，试过了几个随机视频都不可以

**Comment by tianlei2119** [comment]
我也遇到了，1.6.0版也有这个问题

**Comment by yaobiao131** [comment]
我这边测试没有问题，如果一直有问题可以测试下这个版本 https://github.com/yaobiao131/downkyicore

**Reply by Tomshero** [reply]
这个版本可以解析

**Comment by shaoyang0620** [comment]
1.5.9可以正常使用，但是1.6.0和1.6.1都不能解析。

**Reply by Tomshero** [reply]
一样的问题

**Comment by LcxRzh** [comment]
1.59可以解析，但是最高只支持480的，不支持更高的视频。

**Reply by firesiva** [reply]
那是因为1.59版根本就无法登录B站账号

**Comment by JayWang1990** [comment]
同样的问题，新版本无法解析和下载，1.5.8可以

**Comment by shaoyang0620** [comment]
个人解决方法：
1、设置-网络-选择下载器，把“Aria2”改为“自建下载器“。
2、设置-基本-取消“视频自动解析”，取消“解析后自动下载已解析视频“。
此方法已在win10和win11系统下验证可行，但不代表每种不能解析的情况都可以解决，另外，不能下载时，尽量不挂VPN，

**Reply by firesiva** [reply]
我也是win10
感觉VPN确实有影响；不过以前的版本没有影响；
另外，同一个视频不是一直不能解析，而是有时候能解析，有时候不呢

**Reply by Adastra99** [reply]
哇！！！改完下载器就可以用了！谢谢！！！
